molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/service/MappingService.java

```diff
@@ -88,6 +88,7 @@ long applyMappings(
    * @param clonedMappingProjectName name of the cloned {@link MappingProject}
    * @return cloned {@link MappingProject}
    */
+  @SuppressWarnings("unused")
```

Comment:
I think this is a valid warning and should not be suppressed. Instead the code calling this method should at least do something with id of the cloned project (not as part of this PR).